l10n_it_vat_registries/vat_registry.py

```diff
@@ -235,6 +232,18 @@ def get_tax_by_tax_code(self, tax_code_id):
                 else:
                     if tax.id not in tax_ids:
                         tax_ids.append(tax.id)
+        # Nel caso in cui due imposte (una con iva inclusa nel prezzo e 
+        # una con iva esclusa dal prezzo), utilizzino lo stesso account 
+        # tax code o lo stesso account base code, verrà utilizzata solo 
+        # quella con iva esclusa dal prezzo.
+        if len(tax_ids) >1:
+            tax_ids_temp = []
+            for tax in obj_tax.browse(self.cr, self.uid, tax_ids):
+                if not tax.price_include:
+                    tax_ids_temp.append(tax.id)
+            tax_ids = tax_ids_temp
+
+
         if len(tax_ids) != 1:
```

Comment:
In questo caso il controllo deve essere sull'esistenza di un solo elemento in tax_ids, zero o maggiore di uno devono sollevare un'eccezione.